tests/test_environments/k8s-cloudbeat-tests/templates/cloudbeat-eks-pytest.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.cloudbeatEks.test_environments -}}
+{{- if .Values.cloudbeatEks.deploy -}}
```

Comment:
Is this intended?

Reply:
Yes, this fix is the correct one, taken from the Helm values.